go.mod

```
@@ -70,6 +70,22 @@ require (
 	google.golang.org/grpc v1.39.0
 )
 
+// self managed
+require (
+	// snappy: updated for go1.17 support
+	github.com/golang/snappy v0.0.4-0.20210608040537-544b4180ac70 // indirect
```

Note that `go.mod` allows additional comments to be put after `// indirect` by adding a semi-colon (`;`) followed by the comment.

Perhaps we should use that notation so that the comment explaining why we have the override is there, e.g.;

```suggestion
	github.com/golang/snappy v0.0.4-0.20210608040537-544b4180ac70 // indirect; snappy: updated for go1.17 support
```

---

Looks like we might be able to revert the snappy change https://github.com/golang/snappy/pull/61#issuecomment-870732021 but can be separate from here and I haven't verified myself. Seems like the breakage may have only been in rc releases.